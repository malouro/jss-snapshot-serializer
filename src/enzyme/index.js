import enzyme from './enzyme-jss-snapshot-serializer';
export default enzyme;