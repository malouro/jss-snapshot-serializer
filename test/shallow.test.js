import React from 'react';
import { shallow } from 'enzyme';
import MockButton from './mock-components/MockButton';

describe('Shallow Rendering', () => {
  test('serializes `react-jss` component', () => {
    const wrapper = shallow(<MockButton />);

    expect(wrapper).toMatchSnapshot();
  });
});
