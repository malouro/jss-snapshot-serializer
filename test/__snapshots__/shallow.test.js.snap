// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Rendering serializes \`react-jss\` component 1`] = `ShallowWrapper {}`;
